django/tests/otto/test_cleanup.py

```diff
@@ -300,3 +305,120 @@ def test_delete_empty_chats_task(client, all_apps_user):
     # Check that the too-new empty chat remains
     too_new_empty_chat = Chat.objects.filter(id=too_new_empty_chat_id).first()
     assert too_new_empty_chat is not None
+
+
+@pytest.mark.django_db(transaction=True)
+def test_delete_text_extractor_files_task(client, all_apps_user):
+
+    # Ensure the "Otto admin" group exists
+    group, created = Group.objects.get_or_create(name="Otto admin")
```

Comment:
The "Otto admin" group needed to be manually created to add the permissions. This is why the test was failing.